docs/ide/tutorial-csharp-console.md

```diff
@@ -38,7 +38,7 @@ First, we'll create a C# application project. The project type comes with all th
 
 If you don't see the **Console App (.NET Core)** project template, you can get it by adding the **.NET Core cross-platform development** workload. You can add this workload in one of the two following ways, depending on which Visual Studio 2017 updates are installed on your machine.
 
-#### Option 1: Use the New Project dialog box
+#### Method 1: Use the New Project dialog box
```

Comment:
@kallerdaller We prefer to use "Option" as it is a term that we use uniformly throughout our content for situations such as this. Respectfully, I decline this edit.